lib/editing/editor_codemirror.css

```diff
@@ -177,6 +177,17 @@ li.CodeMirror-hint-active {
   background: radial-gradient(#fff, #eee);
 }
 
+.type-quick_fix {
+  font-family: "Inconsolata", sans-serif;
+  font-size: 12pt;
+}
+
+.type-quick_fix::before {
+  content: " ";
+  background: url('../images/wrench.svg');
```

Comment:
Can you move this svg into the same folder as the css file? I think that'll prevent issues with the deployed build.